### PR TITLE
TECEXT-339 Fix for new admin UI

### PR DESCRIPTION
**Pull request by andrasguseo**
[TECEXT-339]

The extension settings stopped showing with the new admin UI.
This PR aims to fix that.

Screenshot:
https://1drv.ms/i/s!Aqh6lbOjUW1ThaotqyQCKyco3kDQ7Q?e=gedeHu

[TECEXT-339]: https://stellarwp.atlassian.net/browse/TECEXT-339?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -33,8 +33,20 @@ Please visit our [extension library](https://theeventscalendar.com/extensions/)
 
 We're always interested in your feedback and our [premium forums](https://theeventscalendar.com/support-forums/) are the best place to flag any issues. Do note, however, that the degree of support we provide for extensions like this one tends to be very limited.
 
+== Upgrade Notice ==
+= [2.1.0] =
+
+This extension is no longer compatible with versions of Events Calendar Pro 7.1.0.
```
Comment (Camwyn) [inline readme.txt]:
```suggestion
This extension is no longer compatible with Events Calendar Pro prior to version 7.1.0.
```

Comment (Camwyn) [reply readme.txt]: missed the "prior" 😉

**Review by Camwyn — CHANGES_REQUESTED** [review]
Small docblock tweak

**Review by bordoni — APPROVED** [review]

File: readme.txt (hunk 1 above)
Comment (Camwyn) [reply readme.txt]: Or:
```suggestion
This extension is no longer compatible with versions of Events Calendar Pro prior to 7.1.0.
```

**Review by Camwyn — APPROVED** [review]

**Pull request merged**